openedx/features/course_experience/__init__.py

```diff
@@ -72,7 +72,16 @@
     __name__,
 )
 
-# Waffle flag to enable relative dates for course content
+# .. toggle_name: course_experience.relative_dates
+# .. toggle_implementation: ExperimentWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to enable relative dates for course content. A 'Dates' tab will be visible in the
+#   course view showing key course dates.
```

Comment:
@BbrSofiane I think the indentation for this newline is not correct, could you please other examples?

Reply:
Hey, are you sure? The other annotations on this file have 3 spaces too.